packages/ingest-mongodb-public/package.json

```diff
@@ -48,16 +49,23 @@
     "typescript": "^5"
   },
   "dependencies": {
+    "@mozilla/readability": "^0.5.0",
```

Comment:
note: check to make sure you're using all the dependencies here. there might be some from when i was hacking around in the demo PR